.gitignore

```diff
@@ -46,7 +46,7 @@ tsan/
 asan/
 ubsan/
 opt/
-build/
+build*/
```

Comment:
This is fine by me, but I am curious, what are you calling your build directories?